wildfly/connectors/file/feature-pack-file/src/main/resources/modules/system/layers/dv/org/jboss/teiid/resource-adapter/s3/main/module.xml

```diff
@@ -0,0 +1,29 @@
+<module xmlns="urn:jboss:module:1.8" name="org.jboss.teiid.resource-adapter.s3">
+    <properties>
+        <property name="jboss.api" value="private"/>
+    </properties>
+       
+    <resources>
+        <artifact name="${org.teiid.wildfly.connectors:connector-s3}"/>
+        <artifact name="${org.teiid.connectors:file-s3}"/>
+        <artifact name="${com.amazonaws:aws-java-sdk-s3}"/>
+        <artifact name="${com.amazonaws:aws-java-sdk-core}"/>
+        <resource-root path="."/>
+        <!-- Insert resources here -->
+    </resources>
+
+    <dependencies>
```

Comment:
What all are these dependencies?

Reply:
Modules are a separate system from poms you need to list any transitive dependencies, teiid dependencies, and even some java dependencies that are typically implicit.